### PR TITLE
Remove minor usage deps

### PR DESCRIPTION
**Pull request by ciaranschutte**
**Description of changes**

remove some packages with minor usage

**Type of Change**

- [ ] Bug
- [ ] Styling
- [ ] New Feature

**Checklist before requesting review:**

- Design (select one):
  - [ ] Matches design:
    - component sizes, spacing, and styles
    - font size, weight, colour
    - spelling has been double checked
  - [ ] No design provided, screenshot of changes included in PR
  - [ ] No changes to UI
- UI Kit (select one):
  - [ ] New Components with storybook stories created
  - [ ] Modified Existing components and updates stories
  - [ ] No new UIKit components or changes
- [ ] Feature is minimally responsive
- [ ] Manual testing of UI feature
- [ ] Add copyrights to new files
- [ ] Connected ticket to PR

### CODE REVIEW

File: src/AppBar/styledComponents.tsx [hunk 1]
```diff
@@ -17,21 +17,23 @@
  * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import React from 'react';
```
[the remark this answers is not in the document]
Comment (ciaranschutte) [reply src/AppBar/styledComponents.tsx:20]: nice catchty

File: src/Modal/index.tsx [hunk 2]
```diff
@@ -105,7 +104,7 @@ const ModalOverlay = styled('div')<{ theme?: any }>`
   display: flex;
   justify-content: center;
   align-items: center;
-  background: ${({ theme }) => Color(theme.uikit.colors.primary_dark).alpha(0.8).hsl().string()};
+  background: hsla(230, 50%, 9.4%, 0.8);
```
Comment (caravinci) [inline src/Modal/index.tsx:107]: ooh fancy! is that a new js thing?

Comment (ciaranschutte) [reply src/Modal/index.tsx:107]: just css :)

Comment (ciaranschutte) [reply src/Modal/index.tsx:107]: true frontend devs dont even css anymore :p

**Review by caravinci — APPROVED** [review]
LGTM 🚀

**Pull request merged**